test/robustness/overlay/buffer/recursive_polygons_buffer.cpp

```diff
@@ -213,7 +216,7 @@ bool test_buffer(MultiPolygon& result, int& index,
     bg::strategy::buffer::end_round end_strategy;
     bg::strategy::buffer::point_circle point_strategy;
     bg::strategy::buffer::side_straight side_strategy;
-    bg::strategy::buffer::join_round join_round_strategy(32); // Compatible with MySQL
+    bg::strategy::buffer::join_round join_round_strategy(settings.points_per_circle);
```

Comment:
what does "compatible with MySQL" means and why it is removed ?

Reply:
It's not removed, and 32 still the default.  But it's flexible now.
It means that MySQL also uses 32 points per circle - but actually for this test that is not very relevant so I removed that comment here.